### ISSUE TITLE
template Errors

### ISSUE THREAD

**Issue by mdengusiak** [ladybug-tools/dynosaur#9]
We have 50 rooms 6 does not work
I will see what there is sth they have in common

we have spaces from 0..49

problematic are 
1. warning Taceback Line 42:
45

2. crushing Revit are:
24,36,43, 47,48

**Comment by mostaphaRoudsari** [comment]
The warning is about the order of vertices and there are ways to get it fixed. I'm not sure about the number 2. Will take a look after Thursday!

**Comment by mdengusiak** [comment]
I just realized one issue... that we might be copying errors from revit space egnine...

this is correct set up, floors walls
![image](https://user-images.githubusercontent.com/14005893/31437724-264437e8-ae7e-11e7-92a1-25748bbe3890.png)

![image](https://user-images.githubusercontent.com/14005893/31437497-5ea9813e-ae7d-11e7-8a87-dc24f39aeb66.png)

however when we have air gaps which means error in Revit 

![image](https://user-images.githubusercontent.com/14005893/31437680-ff2315e4-ae7d-11e7-952c-016ffa4c2cde.png)

![image](https://user-images.githubusercontent.com/14005893/31437529-7b72e148-ae7d-11e7-9c2e-bc7fa633dcf2.png)

**Comment by mostaphaRoudsari** [comment]
Hi @mdengusiak, is this what makes the crash? or you mean AirWall is not recognized by Dynosaur.

**Comment by mdengusiak** [comment]
yes....as you can see when we extract space data from revit ....we do not have already second Floor which is ceiling...there is air-gap in revit analytical model and therefore we can not see second floor. This means we already getting model with issues unfortunately...not sure how we should deal with it
lets talk about this tomorrow
